src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
-// write code here
+const list = document.body.querySelector('ul');
+
+class Employee {
+  constructor(boss, position, salary, age) {
+    this.name = boss;
+    this.position = position;
+    this.salary = salary;
+    this.age = age;
+  }
+}
+
+function sortList() {
+  const employeesArr = [ ...list.children ];
+  let resultString = '';
+
+  employeesArr.sort((a, b) => {
+    return Number(b.dataset.salary.substring(1).replaceAll(',', ''))
```

Comment:
move `salary.substring(1).replaceAll(',', '')` this logic to the separate function